includes/class-newspack-popups-inserter.php

```diff
@@ -653,6 +653,10 @@ public static function enqueue_scripts() {
 			\wp_enqueue_style( $admin_script_handle );
 		}
 
+		if ( self::assess_has_disabled_popups() ) {
+			return;
+		}
+
```

Comment:
I think this breaks the solution from #1436 and prevents pageviews from being logged on pages without prompts.

Reply:
Is criteria.js being enqueued causing any issues?